File: fizzbuzz.c
#include <stdio.h>

void fizzbuzz(int n)
{
     if (n % 15 == 0) printf("FizzBuzz\n");
     else if (n % 3 == 0) printf("Fizz\n");
     else if (n % 5 == 0) printf("Buzz\n");
     else printf("%d\n", n);
}

int main(void)
{
     for (int i = 1; i <= 100; i++) {
          fizzbuzz(i);
     }
}

File: fizzbuzz.lua
local function fizzbuzz(n)
   if n % 15 == 0 then print("FizzBuzz")
   elseif n % 3 == 0 then print("Fizz")
   elseif n % 5 == 0 then print("Buzz")
   else print(n)
   end
end

for i = 1, 100 do
   fizzbuzz(i)
end
